app/code/Magento/Quote/Model/QuoteManagement.php

```diff
@@ -456,6 +456,7 @@ protected function submitQuote(QuoteEntity $quote, $orderData = [])
             $order->setShippingAddress($shippingAddress);
             $order->setShippingMethod($quote->getShippingAddress()->getShippingMethod());
         }
+        $order=$this->quoteAddressToOrder->convert($quote->getBillingAddress(), $orderData);
```

Comment:
Why do you redefine `$order` here? Is there any problem with Billing Address conversion happening below?